Automattic/simplenote-ios
# Update Fastlane and Sentry gems

This PR makes Fastlane happy updating it and the Sentry plugin. 

### Test
1. Verify that this PR is green.
2. Run `bundle install` in your local machine and verify that it succeeds.
3. Checkout a test branch from the current one, bump the version in `Version.Public.xcconfig` and commit.
4. Run `bundle exec fastlane build_and_upload_internal` and verify that the build succeeds and that the symbols are uploaded to Sentry.
5. Remove the test build from AppCenter. 

### Review
 Only one developer is required to review these changes, but anyone can perform the review.

### Release
These changes do not require release notes.
